Toontown_Tower_Defense.pde

```diff
@@ -420,5 +496,6 @@ void mouseClicked()
         gagSelected = false;
       }
     }
+    */
   }
 }
```

Comment:
It's customary to end files with a new line. You can either change that now or in a future commit.

Reply:
I meant to comment on line 501.